zeeguu/api/endpoints/translation.py

```diff
@@ -40,7 +41,7 @@ def get_one_translation(from_lang_code, to_lang_code):
 
     :return: json array with translations
     """
-
+    IS_DEV_SKIP_TRANSLATION = int(os.environ.get("DEV_SKIP_TRANSLATION", 0)) == 1
```

Comment:
should we move this at the top of the file? otherwise we're calling it every time we call the method. and it should not be needed: the envvars are unlikely to change during the running of the process.

Reply:
the fact that it's all uppercase makes it even more ready to be a constant at the top :)

Reply:
Indeed, I am not sure why it was in the method itself. I have moved it now!